workers/generic-worker/tc/services.go

```diff
@@ -0,0 +1,51 @@
+package tc
+
+import (
+	"net/url"
+	"time"
+
+	"github.com/taskcluster/taskcluster/v30/clients/client-go/tcauth"
+	"github.com/taskcluster/taskcluster/v30/clients/client-go/tcpurgecache"
+	"github.com/taskcluster/taskcluster/v30/clients/client-go/tcqueue"
+	"github.com/taskcluster/taskcluster/v30/clients/client-go/tcsecrets"
+	"github.com/taskcluster/taskcluster/v30/clients/client-go/tcworkermanager"
+	"github.com/taskcluster/taskcluster/v30/workers/generic-worker/tclog"
+)
+
+type Auth interface {
```

Comment:
This seems to be the same pattern used in the worker-runner tests.  It would be good to have similar explanatory comments as to what the interfaces are for.
```
// An interface containing the functions required of WorkerManager, allowing
// use of fakes that also match this interface.
```

Reply:
I've added as a package comment.